include/avif/internal.h

```diff
@@ -89,9 +89,9 @@ typedef float (*avifTransferFunction)(float);
 // and can go beyond 1.0 for HDR transfer characteristics:
 // - For AVIF_TRANSFER_CHARACTERISTICS_SMPTE2084 (PQ), the linear range is [0.0, 10000/203]
 // - For AVIF_TRANSFER_CHARACTERISTICS_HLG, the linear range is [0.0, 1000/203]
-avifTransferFunction avifTransferCharacteristicsGetGammaToLinearFunction(avifTransferCharacteristics atc);
+AVIF_API avifTransferFunction avifTransferCharacteristicsGetGammaToLinearFunction(avifTransferCharacteristics atc);
```

Comment:
We should not export the private functions declared in internal.h. Even though we don't provide the internal.h header to our clients, they can still call these functions if these functions are exported.

Note: In some Unix platforms (including Linux), we can give exported symbols a version (which is a string), which allows us to mark certain exported functions as private exports by using a version with "Private" in its name. But I don't think this kind of symbol versioning feature is available on Windows, and it may not even be available on all Unix platforms.